etcdmain/etcd.go

```diff
@@ -108,6 +113,27 @@ func startEtcdOrProxyV2() {
 		plog.Warningf("no data-dir provided, using default data-dir ./%s", cfg.Dir)
 	}
 
+	// The global tracer is noop by default.
+	if cfg.TracingLightstepToken != "" {
```

Comment:
This is in my opinion one of the most important parts about opentracing: You don't have to modify the entire codebase to use a different tracing backend. It's a quick change in the configuration and everything else stays the same.